packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -78,7 +78,7 @@ class SlideshowView extends GalleryComponent {
       return false;
     }
 
-    if (this.container) {
+    if (this.scrollElement) {
```

Comment:
This condition is duplicated inside `this.getScrollLeft()`, therefor the same "work" (checking `this.scrollElement` is being done twice :(

Reply:
Since `this.getScroolLeft` returns 0 when `this.scrollElement` is `undefined` you can use that function to `return false` (instead of duplicating the tests:
```
isScrollStart() {
    return this.props.styleParams === GALLERY_CONSTS.slideAnimations.SCROLL) || this.getScrollLeft() > 1
}
```